src/utils/components/SSHAccess/components/VirtctlSSHCommandClipboardCopy.tsx

```diff
@@ -21,21 +20,13 @@ const VirtctlSSHCommandClipboardCopy: FC<VirtctlSSHCommandClipboardCopyProps> =
   }
 
   return (
-    <>
-      <ClipboardCopy
-        clickTip={t('Copied')}
-        data-test="ssh-over-virtctl"
-        hoverTip={t('Copy to clipboard')}
-      >
-        {getConsoleVirtctlCommand(vm)}
-      </ClipboardCopy>
-      <HelperText className="pf-u-mt-sm">
-        <HelperTextItem variant="indeterminate">
-          {t('Example: ')}
-          {getConsoleVirtctlCommand(vm, exampleIdentityFilePath)}
```

Comment:
Can you use this example instead of hard coding the command in the popover?

Reply:
If I'm understanding Guohua's comment on the jira issue correctly this is how it's supposed to be. I believe the goal is to just have a static example.